package.json

```diff
@@ -2,5 +2,8 @@
   "name": "badminton-games",
   "license": "MIT",
   "version": "2.0.0",
-  "workspaces": ["packages/*", "app"]
+  "workspaces": ["packages/*", "app"],
+  "scripts": {
+    "lint": "cd ./app/ && bun run lint"
```

Comment:
This is an `app` specific command, it's probably best to leave it just as part of it's `package.json`.

Reply:
That's fair, it was a combined run them both command, but you're probs only using one or the other, and the pipeline still does both